.github/workflows/beacon-genesis.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow initiates Beacon genesis on Testnet. The workflow should be
+# manually dispatched after the `keep-core` contracts get deployed and
+# kubernetes pods get rotated. The `upstream_build` parameter should contain
+# deployment info of at least `keep-core/solidity` module, but inputs
+# additionally containing information about other modules will be accepted as well.
+name: Beacon genesis
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment for workflow execution'
+        required: false
+        default: 'dev'
+      upstream_builds:
+        description: 'Upstream builds'
+        required: false
+
+jobs:
+  beacon-genesis:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./solidity/scripts
```

Comment:
Can we operate from `./solidity`? It's the npm module root (and Truffle too) and it looks better when `npm install` and friends are invoked here than in a subdirectory. It's ok to have `npx truffle exec ./scripts/genesis.js —network ${{ env.TRUFFLE_NETWORK }}` in case we apply this suggestion. We also use this approach across other repos.

Reply:
Agree. Applied in cd650a3654b3d71f94d4d308176ba629c15a7eae.